src/core/census/grpc_context.c

```diff
@@ -40,6 +41,7 @@ static void grpc_census_context_destroy(void *context) {
 }
 
 void grpc_census_call_set_context(grpc_call *call, census_context *context) {
+  GRPC_API_TRACE("grpc_census_call_set_context(%p, %p)", 2, (call, context));
```

Comment:
suggest adding arg names everywhere:

`grpc_census_call_set_context(call=%p, context=%p)`

since it makes these logs way more usable for debugging without needing to cross reference all over the place